index.html

```diff
@@ -8,50 +8,50 @@
     <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-    <top_nav>
-        <item_nav>
-            <frame4>
-                <portfolio>portfolio</portfolio>
-            </frame4>
-            <frame3>
-                <about>About</about>
-            </frame3>
-            <frame5>
-                <contact>Contact</contact>
-            </frame5>
+    <to1_nav1 id="to1_nav1">
+        <item_nav id="item_nav">
+            <fram4>
+                <portf id="portf">portfolio</portf>
+            </fram4>
+            <fram3>
+                <about id="about">About</about>
+            </fram3>
+            <fram5>
+                <contact id="contact">Contact</contact>
+            </fram5>
         </item_nav>
 
-        <name>
-            <johndoe>John doe</johndoe>
-        </name>
+        <nam id="nam">
+            <johndoe id="johndoe">John doe</johndoe>
+        </nam>
 
-        <rectangle></rectangle>
+        <rectang id="rectang"></rectang>
 
-        <normal_btn>
-            <icons>
-                <union1>
+        <normal_btn id="normal_btn">
+            <icons id="icons">
+                <union1 id="union1">
+                    <vectorx id="vectorx">
                     <vectorx>
-                    <vectorx>
-                    <rec>
+                    <rec id="rec">
                         <img src="./assets/Icon - Mail.png" alt="">
                     </rec>
                 </union1>
             </icons>
         </normal_btn>
 
-        <hamburger>
+        <hamburger id="hamburger">
             <img src="./assets/Enabled.svg" alt="">
         </hamburger>
-    </top_nav>
+    </to1_nav1>
 
 
 
 
-    <social_icons>
-        <medium>
+    <social_icons id="social_icons">
+        <medium id="medium">
             <img src="./assets/medium.svg" alt="">
         </medium>
-        <github>
+        <github id="github">
```

Comment:
- [ ] Kindly I recommend you to use `header` tag for Toolbar section, Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.
- [ ] A stated the project requirement You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So for this activity you are required to build the **first 2 sections of the mobile website using the template you chose** But I see in your Project for toolbar section you design desktop toolbar so kindly change design for mobile website just like the figma design .

The Figma toolbar section Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/205013504-819fe215-e4cb-444f-95e1-65dd1f25e986.PNG)

Your Toolbar section design look likes this : Mobile website(Minimum mobile screen size should be 375px) kindly work on it .

![Capture](https://user-images.githubusercontent.com/93598189/205013969-eb2e06c9-74ad-45a9-bc29-79da1d3061d6.PNG)

![Capture](https://user-images.githubusercontent.com/93598189/205014177-823bd239-e19e-40d9-9fc4-376616f9f3c5.PNG)

Reply:
Did not know I had to use a picture for mobile UI elements. fixed. to1_nav changed to header as requested